apps/services/routers.py

```diff
@@ -262,6 +262,29 @@ async def read_request_details(
     }
 
 
+@services_router.post("/{university_id}/create-students/",
+                      name="create_students_list_from_file",
+                      response_model=JSENDOutSchema[Union[List[CreateStudentsOut], None]],
```

Comment:
Or we can't so that because of

"Can't perform output without newly created CreateStudentsOut, and there is no ability to use FullNameSchema"

Reply:
Changed to CreateStudentsListOut

Reply:
resolved